src/omega_edit/client.ts

```diff
@@ -45,19 +45,21 @@ async function getOmegaEditPort(port: number | undefined = undefined) {
     const defaultServerPort = vscode.workspace
       .getConfiguration('dataEditor')
       .get<number>('defaultServerPort', serverPort)
-    const portEntered = await vscode.window.showInputBox({
-      prompt: 'Enter port number to run omega-edit server on',
-      value: defaultServerPort.toString(),
-    })
 
-    if (portEntered) {
-      serverPort = parseInt(portEntered)
-    } else {
-      serverPort = 0
-      throw Error('Bad port entered')
-    }
+    serverPort = defaultServerPort
```

Comment:
Update function `getOmegaEditPort` to be

```ts
async function getOmegaEditPort() {
  const defaultServerPort = vscode.workspace
    .getConfiguration('dataEditor')
    .get<number>('serverPort', serverPort)

  serverPort = defaultServerPort
}

```

This will always assign a port.